src/main/api/2/studio-api-2.yaml

```diff
@@ -6920,6 +6920,307 @@ paths:
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+
+  /workflow/packages/{id}:
+    get:
+      tags:
+        - workflow
+      summary: Get workflow package by id
+      description: Required permission "requst_publish, publish"
+      operationId: workflowGetPackage
+      parameters:
+        - name: id
+          in: path
+          description: The workflow package ID
+          required: true
+          schema:
+            type: string
+            format: UUID                
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  response:
+                    $ref: '#/components/schemas/ApiResponse'
+                  package:
+                    $ref: '#/components/schemas/WorkflowPackage'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+
+  /workflow/packages/approve:
+    post:
+      tags:
+        - workflow
+      summary: Approve workflow package
+      description: Required permission "publish"
+      operationId: workflowApprovePackage
+      requestBody:
+        description: parameters to approve a workflow package
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                siteId:
+                  type: string
+                  description: site identifier
+                packageId:
+                  type: string
+                  description: Id of the workflow package to approve
+                schedule:
+                  type: string
+                  format: datetime
+                  description: approved scehedule when to publish package
+                approverComment:
+                  type: string
+                  description: approver's comment about approved workflow package
+              required:
+                - siteId
+                - packageId
+      responses:
+        '201':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  response:
+                    $ref: '#/components/schemas/ApiResponse'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+
+  /workflow/packages/reject:
```

Comment:
`/package`

Reply:
I took groups and users API as a guideline. We have plural for namespace there.